mypy/__main__.py

```diff
@@ -5,8 +5,14 @@
 
 
 def console_entry() -> None:
-    main(None, sys.stdout, sys.stderr)
+    try:
+        main(None, sys.stdout, sys.stderr)
+    except BrokenPipeError:
+        sys.exit(2)
 
 
 if __name__ == '__main__':
-    main(None, sys.stdout, sys.stderr)
+    try:
+        main(None, sys.stdout, sys.stderr)
+    except BrokenPipeError:
+        sys.exit(2)
```

Comment:
Why not just call console_entry()? That saves ugly repetition.

Reply:
I wondered about this as well but thought it was there for a specific reason. But that should probably not hurt anything